conf/distro/include/raspberrypi_generic_dunfell.conf

```diff
@@ -1,3 +1,8 @@
+PACKAGECONFIG_append_pn-wpeframework_dunfell = " ${@bb.utils.contains('DISTRO_FEATURES', 'container', 'processcontainers processcontainers_lxc', '', d)}"
+PACKAGECONFIG_append_pn-wpeframework-plugins_dunfell = " ${@bb.utils.contains('DISTRO_FEATURES', 'container', 'processcontainers', '', d)}"
+
+CMDLINE_append_pn-linux-raspberrypi_dunfell = " ${@bb.utils.contains('DISTRO_FEATURES', 'container', 'cgroup_enable=memory cgroup_memory=1', '', d)}"
+
 PREFERRED_VERSION_gstreamer1.0-omx_dunfell = "1.16.%"
-PREFERRED_VERSION_linux-raspberrypi_dunfell = "4.19.%"
+PREFERRED_VERSION_linux-raspberrypi_dunfell = "5.10.%"
```

Comment:
How about kernel version for other yocto version? Shouldn't kernel version be global for all yocto versions?

Reply:
Since each yocto flavour has different kernel version support,  so not sure how to make same for all yocto version. For dunfell I upgrade to 5.10 based on yocto master flavour.